nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py

```diff
@@ -123,7 +123,7 @@ def generate_launch_description():
     # Define commands for launching the navigation instances
     nav_instances_cmds = []
     for robot in robots:
-        params_file = LaunchConfiguration(robot['name'] + '_params_file')
+        params_file = LaunchConfiguration(f"{robot['name']}_params_file")
```

Comment:
I would probably try to keep single / double quotes consistent across the code base. If not for any other specific reason, this discussion could be valuable https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python (but my personal preference is just to make sure to be consistent, given there are no double / single quotes in the text itself)

Reply:
Oh yeah, @timple make sure you're running the linters and this passes in the python ones

Reply:
@klintan: Agreed on the consistency, changed.

@SteveMacenski: I doubt these kind of CI errors are related to the python strings?
```
/opt/overlay_ws/src/navigation2/nav2_dwb_controller/dwb_critics/test/obstacle_footprint_test.cpp:246
Expected equality of these values:
  critic->lineCost(3, 3, 0, 50)
    Which is: 85
  50
```

`ament_pep257` succeeds locally.

Reply:
No, other issues, have been fixed now but your fork is probably outdated. Merging! Thanks!